libs/causality/src/lib/CausalAnalysisDashboard/Controls/CausalAnalysisView/CausalIndividualView/CausalIndividualChart.tsx

```diff
@@ -458,6 +458,40 @@ export class CausalIndividualChart extends React.PureComponent<
         trace.y = rawY;
       }
     }
+    if (
```

Comment:
nit:
line 461-467 can be updated to
`const identityFeatureName = this.context.jointDataset.datasetMetaData.featureMetaData?.identity_feature_name;` So we can reduce one layer of if condition statement without logic change